build-tests/rush-redis-cobuild-plugin-integration-test/sandbox/sharded-repo/projects/e/config/rush-project.json

```diff
@@ -0,0 +1,13 @@
+{
+  "operationSettings": [
+    {
+      "operationName": "_phase:build",
+      "outputFolderNames": ["dist"],
+      "shards": 100
+    },
+    {
+      "operationName": "cobuild",
+      "outputFolderNames": ["dist"]
+    }
+  ]
+}
```

Comment:
## Design idea 1 discussed today:

**package.json**
```js
{
  "scripts": {
    "_phase:build": "invoke-jest",
    "_phase:build:after-shards": "./collate-coverage.js",
  }
}
```

**collate-coverage.js**
```js
const shardCount = parseInt(process.env._RUSH_SHARDS_COUNT);
for (let i: number  = 0; i<shardCount; ++i) {
  // Its build cache entry reads from "_phase:build" output folder
 `${process.env._RUSH_SHARDS_FOLDER}/${index}`
}
```

## Design idea 2

Design idea discussed today:

**package.json**
```js
{
  "scripts": {
    // 
    "_phase:build:shard": "invoke-jest",
    "_phase:build": "./collate-coverage.js"
  }
}
```

**collate-coverage.js**
```js
const shardCount = parseInt(process.env._RUSH_SHARDS_COUNT);
for (let i: number  = 0; i<shardCount; ++i) {
  // Its build cache entry reads from "_phase:build" output folder
 `${process.env._RUSH_SHARDS_FOLDER}/${index}`
}
```

Reply:
I am leaning towards the first, this PR uses `_phase:build` and `_phase:build:collate` or `_phase:test` and `_phase:test:collate`. I think that makes more sense as you need to add a new command to reconcile the phases and your existing commands read the same, so `_phase:test` maps to jest and you have a new `_phase:test:collate` command to collate the coverage.